h3/src/quic.rs

```diff
@@ -29,7 +29,7 @@ impl<'a, E: Error + 'a> From<E> for Box<dyn Error + 'a> {
 /// Trait representing a QUIC connection.
 pub trait Connection<B: Buf> {
     /// The type produced by `poll_accept_bidi()`
-    type BidiStream: SendStream<B> + RecvStream;
+    type BidiStream: BidiStream<B>;
```

Comment:
The `BidiStream` trait is an opt-in, in case the QUIC library is able to support splitting them. The goal is that splitting shouldn't be _required_.

Reply:
Thanks, I understood it the other way around :sweat_smile: